src/main/java/machines/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package machines;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer has started the job");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped the job");
```

Comment:
Similarly, the stop message should also be informative and specific to the machine type. Include the class name to clarify which machine has stopped working.